src/main/java/de/tum/in/www1/artemis/repository/StudentExamRepository.java

```diff
@@ -126,6 +126,8 @@ SELECT count(se) FROM StudentExam se
             """)
     Optional<StudentExam> findByExamIdAndUserId(@Param("examId") long examId, @Param("userId") long userId);
 
+    boolean existsByExam_CourseIdAndExamIdAndUserId(@Param("courseId") long courseId, @Param("examId") long examId, @Param("userId") long userId);
```

Comment:
This function could benefit from javadocs, since its intuitively understandable what it does by its name.
Also make sure to keep to the naming convention i.e. no underscores here

Reply:
We have to have the underscore here because they are part of the syntax for Spring JPA Data methods, as also mentioned in [Moritz's comment](https://github.com/ls1intum/Artemis/pull/5665#discussion_r978112006). I will add Javadocs.

Reply:
I added the Javadocs for the method.